### PR TITLE
Split ensemble slides into bagging and boosting

### PR DESCRIPTION
**Pull request by ArturoAmorQ**
This PR adds the slides `bagging.md` and `boosting.md` that result from splitting `ensemble.md`.

Doing so required the refactoring of the quizzes in M6 and modifying the `toc.yml` file accordingly.

It also adds spacing to the tables in the `custom.css` file.

**Note to ourselves:** This PR has to be merged along with the respective gitlab MR.

### CODE REVIEW

**Comment by lesteve** [comment]
Would be it worth doing related things in the same PR or is it better to leave it for a further PR, not sure ...

Here is the kind of things I have in mind:
- remove slides/ensemble.md?
- update the JupyterBook to put the slides in the right place?

File: jupyter-book/ensemble/boosting_slides.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# 🎥 Intuitions on ensemble models: boosting
+
+TODO: insert video player here once ready
+
+<iframe src="../slides/index.html?file=../slides/boosting.md#p1"/>
```
Comment (ogrisel) [inline jupyter-book/ensemble/boosting_slides.md:5]:
```suggestion
```

File: jupyter-book/ensemble/bagging_slides.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# 🎥 Intuitions on ensemble models: bagging
+
+TODO: insert video player here once ready
+
+<iframe src="../slides/index.html?file=../slides/bagging.md#p1"/>
```
Comment (ogrisel) [inline jupyter-book/ensemble/bagging_slides.md:5]:
```suggestion
```

**Comment by ogrisel** [comment]
I started reviewing the private MR with the quiz answers: https://gitlab.inria.fr/learninglab/mooc-scikit-learn/mooc-scikit-learn-coordination/-/merge_requests/36

**Review by ogrisel — APPROVED** [review]
LGTM!

**Pull request merged**